packages/cardano-services/test/Handle/TypeOrmHandleProvider.test.ts

```diff
@@ -38,4 +38,10 @@ describe('TypeOrmHandleProvider', () => {
 
     expect({ handle, hasDatum }).toEqual({ handle: 'TestHandle', hasDatum: false });
   });
+
+  it('fetches all distinct policy ids', async () => {
+    const result = await provider.getPolicyIds();
+    expect(result.length).toBe(1);
+    expect(result).toEqual(['2c74874ae85a596ad88adc391c5fd7fdb8d8f85e22ced233ee85539d']);
```

Comment:
This assertion is a little weak as we can't see the source. Where is this value defined and why do we expect it here?

Reply:
This data is inserted into the database [here](https://github.com/input-output-hk/cardano-js-sdk/blob/98262e0f2861b9e32956c0852c1d23bea3ae9b64/packages/cardano-services/test/jest-setup/snapshots/handle.sql#L1257-L1260). Since they both have the same policyId, the assertion expects just one to be selected.

Reply:
OK, here's two changes that can help improve the assertion:

1. Make a comment: `Sourced from https://github.com/input-output-hk/cardano-js-sdk/blob/98262e0f2861b9e32956c0852c1d23bea3ae9b64/packages/e2e/local-network/scripts/mint-handles.sh#L14`

2. Since the ID isn't part of the test intent:`expect(result.length).toBe(1);`

Reply:
Do you mean I should remove the second assertion `expect(result).toEqual(['2c74874ae85a596ad88adc391c5fd7fdb8d8f85e22ced233ee85539d'])` and leave only `expect(result.length).toBe(1)`

Reply:
The policy ID can't be verified just by looking at the test or source code, so just an assertion on result length is enough IMO. Alternatively you could independently determine the policy ID from the same keys, which would make the test transparent and self-verifying, but linking to the source showing we're only minting under a single policy seems reasonable. Thoughts @mkazlauskas @iadmytro?

Reply:
Agree, verifying length is sufficient here. IF we want to assert the policy id then we can also do this:
```ts
expect(typeof result[0]).toBe('string');
expect(result[0]).toHaveLength(56);
```

Reply:
Agree with both suggestions

Reply:
Fixed on 9cac32d

Reply:
I was referencing the originating source, which is a script in the `e2e` package, but this is fine too.